addons/packages/secretgen-controller/0.7.1/package.yaml

```diff
@@ -3,23 +3,42 @@ kind: Package
 metadata:
   name: secretgen-controller.community.tanzu.vmware.com.0.7.1
 spec:
+  valuesSchema:
```

Comment:
Since this part is generated out of the `schema.yaml` of each package, can we do some automation so that we can avoid manually copy and paste?

Reply:
@blc1996 Step 5 command in this https://github.com/vmware-tanzu/community-edition/blob/732e5cd301bda4b3d9a2e22c1122d43fbd886428/docs/site/content/docs/latest/designs/package-process.md#8-generate-openapiv3-schema-and-embed-it-in-a-package, is used to generate the above section so it is automated and not manually added